actionpack/test/controller/parameters/nested_parameters_permit_test.rb

```diff
@@ -12,6 +12,17 @@ def assert_filtered_out(params, key)
     params = ActionController::Parameters.new(
       book: {
         title: "Romeo and Juliet",
+        sections: [[{
+          title: "Romance",
+          description: "A Romance section"
+        }]],
```

Comment:
If this array had two sections I'd expect both sections to be permitted but only the first one is.

Also, I think this syntax is ambiguous.

Given `params.permit(sections: [[[:title]]])`, what would be result for the following hashes ?

```ruby
         sections: [
          [
            {
              title: "romance",
              description: "a romance section"
            }
          ],
          [
            {
              title: "drama",
              description: "a drama section"
            }
          ]
        ],
```

And

```ruby
        sections: [
          [
            {
              title: "romance",
              description: "a romance section"
            },
            {
              title: "drama",
              description: "a drama section"
            }
          ]
        ],
```

Reply:
I would permit both